src/index.html

```diff
@@ -1,13 +1,40 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body class="content">
+    <div class="card content__card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="mac"
+        class="card__img"
+      >
+      <div class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+      <div class="card__code">Код товара: 195434</div>
+      <div class="card__rating">
+        <div class="stars">
+          <div alt="star" class="stars__star stars__star--active"></div>
+          <div alt="star" class="stars__star stars__star--active"></div>
+          <div alt="star" class="stars__star stars__star--active"></div>
+          <div alt="star" class="stars__star stars__star--active"></div>
+          <div alt="star" class="stars__star" ></div>
+        </div>
```

Comment:
add the ability to change active stars count via modifier for stars( look layout_stars)

Reply:
don't use alt for `div`

Reply:
i have just one row, so i can change active stars count like this 
.stars__star:nth-child(-n + (change this number)) {
  background-image: url(./images/star-active.svg);
}

Reply:
done